developer_manual/client_apis/OCS/ocs-api-overview.rst

```diff
@@ -44,24 +44,74 @@ This request returns the available metadata of a user. Admin users can see the i
 			<itemsperpage></itemsperpage>
 		</meta>
 		<data>
-			<enabled>true</enabled>
+			<enabled>1</enabled>
+			<storageLocation>/path/to/storage/location/userid</storageLocation>
+			<id>userid</id>
+			<lastLogin>1578283711000</lastLogin>
+			<backend>Database</database>
+			<subadmin/>
 			<quota>
-				<free>338696790016</free>
-				<used>7438874</used>
-				<total>338704228890</total>
-				<relative>0</relative>
-				<quota>-3</quota>
+				<free>20632824998</free>
+   				<used>842011482</used>
```

Comment:
spaces :)

Reply:
Darn spaces ;)
I think now I've got them all in the API response blocks...